### DISCUSSION TITLE
Possible to get a full report on which rules are enabled?

### DISCUSSION THREAD

**Discussion by bichuga** [dotnet/roslyn#56184] [Q&A] [answered]
Is it possible to get a report on exactly which analyzers enabled for a project? We want to be able to audit which analyzers were enabled.

**Comment by jmarolf** [comment, marked as answer]
if you pass `/p:reportanalyzer=true` to msbuild you will get a report of all analyzers that were run and how long they took.  You can also look at the `Analyzer` item group in any msbuild binlog file (which should be fairly easy to do with the [viewer](https://msbuildlog.com/))
